bikeshed/boilerplate.py

```diff
@@ -305,10 +305,10 @@ def addIndexSection(doc):
         container, E.h2({"class": "no-num no-ref", "id": safeID(doc, "index")}, "Index")
     )
 
-    if len(findAll(config.dfnElementsSelector, doc)):
+    if not findAll(config.dfnElementsSelector, doc):
```

Comment:
All of these conditions appear to be exactly opposite what they should be, yeah?

Reply:
Yeah, I went back and just added a value comparison instead of trying to do a (reversed) truthy